modules/tinymce/src/themes/silver/demo/html/demo/demo.html

```diff
@@ -1,114 +1,596 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no, maximum-scale=1.0">
-<title>silver-theme Demo Page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no, maximum-scale=1.0">
+  <title>silver-theme Demo Page</title>
+  <style type="text/css">
+    .tox-context-bar-layout-transition-animation {
```

Comment:
Should this be going somewhere in oxide instead?

Reply:
I have no idea. It is mainly for demo purposes here?

Reply:
Without this bit of CSS, you'll never get the `transitioned` event being fired on the toolbar for the `top` or `bottom` style properties, meaning that the context bar transition class won't get removed. Maybe we need some design input here to decide, but I feel like this animation either needs to be properly part of the editor (aka inside of oxide) or we should be taking the code for this animation class out.

Reply:
We had a conversation and changed how the class was generated and handled, so it is now more explicit.